src/main/java/ca/ciccc/MagicSquare.java

```diff
@@ -71,8 +71,47 @@ public void setSquare(Integer[][] square) {
      */
     public static boolean isMagicSquare(Integer[][] square) {
         // TODO: You need to implement this method.
+        int answer = 0;
+        int rowTotal = square.length;
 
-        return false;
+        for(int i = 0; i < rowTotal; i++ ){
```

Comment:
"answer" value is always sum of square last value in this loop. So you don't need double for loop.